process/pull_request.md

```diff
@@ -1,28 +1,19 @@
 # Pull request
 
-The following outlines how contributors should make changes to the repo. The goals are
+The following outlines how contributors should make changes to a repo. The goals are
 
 - Focused PR's related to specific changes
 - Code review is simple and manageable
 - A clean tree of changes in the commit history
 - Consistency across the commits
 
-We want to respect the time and effort of those reviewing the work. Keeping PR's small, focused and consistent makes their life easier.
+We also want to respect the time and effort of those reviewing the work. Keeping PR's small, focused and consistent makes their life easier.
 
 ## Always on a branch
 
-No matter how small the change, all changes must be done on a branch and never directly to `master` (or `develop` where [git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) is being used). This is to support the principle of [Master is always shippable](../principles/master_is_always_shippable.md)
+No matter how small the change, all changes should be done on a branch and never directly to `master`. This is to support the principle of [Master is always shippable](../principles/master_is_always_shippable.md)
 
-Clone the repo then create your new branch. Please use one of the following prefixes to help categorise the change being made
-
-- `feature` (you're adding new functionality to the service or project)
-- `fix` (you're fixing an issue that affects the overall service, or an issue with the project)
-
```

Comment:
It would be good to just put something into the PR explaining about the removal of these prefixes from the process.

I don't disagree with it, it's just that it might be helpful for people to understand the implications for the handling of hot fixes.

Reply:
That's a good point. Hopefully its enough to pick it up here, so here's my reasoning.

1. **The feature/fix prefixes didn't fit all scenarios**. Tweaking content, adding additional detail to a README etc, could be described as a `feature` but its not really in keeping with the original definition.
1. **Branches are lost from the commit history**. As we now guide people to use the squash and merge option rather than just merging, the fact you had a branch is gone from the commit history. In this context the name of the branch has less value, particularly when you look at a visual representation
1. **Unnecessary ceremony**. It was intended to be meaningful at the time, but in practice it doesn't really matter what the branch name is. Therefore requiring people to select a prefix and then add it to their branch name just becomes unnecessary ceremony
1. **Labels are better**. Using [labels](https://help.github.com/articles/about-labels/) to categorise PR's has more value and opens more options. For example if you stick to using the default GitHub labels and remember to assign them to your PR's, this information can be used when generating your CHANGELOG (see below) 
<img width="591" alt="screen shot 2018-09-28 at 09 18 16" src="https://user-images.githubusercontent.com/1789650/46196622-85151300-c2ff-11e8-87d7-d9e6780667e0.png">

So for those that come afterwards, I hope this explains why I decided to drop using branch prefixes from the guide.

Reply:
Gone through this with @bensagar-ea and what I've recorded here is sufficient. With no other objections I'm merging this 🐶!